wandb/sdk/internal/sender.py

```diff
@@ -1512,6 +1512,7 @@ def _send_artifact(
             client_id=artifact.client_id,
             sequence_client_id=artifact.sequence_client_id,
             metadata=metadata,
+            ttl_duration_seconds=artifact.ttl_duration_seconds or None,
```

Comment:
This means that `None` and `0` are treated the same. This is probably fine, but we should document it in the `ttl` property.

Reply:
i'll have it so you can't set ttl = 0 with a warning of Artifact TTL duration has to be positive

Reply:
Sounds good. Let's add a comment here saying that TTL will never be 0, so that someone looking at this code will know it's fine to treat 0 the same as `None`.